docs/rules.md

```diff
@@ -0,0 +1,12 @@
+# Rules of karmada
+
+## Reserved namespaces
+
+Karmada takes some namespaces  as reserved namespace in order to make karmada work healthily. Resources under these namespaces will not be propagated by default. The reserved namespace list shows as following,
+
+- karmada-system
+- karmada-cluster
+- karmada-es-*
+- kube-system
+- kube-public
+- kube-node-lease
```

Comment:
```suggestion
# Reserved Namespaces

> Note: Avoid creating namespaces with the prefix `kube-` and `karmada-`, since they are reserved for Kubernetes
> and Karmada system namespaces.

For now, resources under the following namespaces will not be propagated:
- namespaces prefix `kube-`(including but not limited to `kube-system`, `kube-public`, `kube-node-lease`)
- karmada-system 
- karmada-cluster
- karmada-es-*
```
Re-organized the contents.

Reply:
Ok, done!